.ado/variables/vars.yml

```diff
@@ -1,5 +1,5 @@
 variables:
   VmImageApple: macOS-11
-  VmImageUbuntu: ubuntu-20.04
+  PoolUbuntu: cxe-ubuntu-20-04-small
```

Comment:
Does `pool` have a semantically better meaning than just `vmImage`? And could we perhaps add a comment to this variable specifying this is a "compliant" pool to explain why we aren't just using `ubuntu-latest` / `ubuntu-20.04` ?

Reply:
If you don't specify a pool name it is implied to be the Microsoft Hosted  pool. and the vmimage is a subfield.
With 1ES Hosted pools you specify the pool. To specialize between images on that pool you could use demands. But I've configured it as 1:1.
Will add a comment...

Reply:
no need for   - group: InfoSec-SecurityResults
  - name: BUILDSECMON_OPT_IN ?

Reply:
For ubuntu not, that is only for mac.